### DISCUSSION TITLE
Has the playwright addon been abandoned?

### DISCUSSION THREAD

**Discussion by bensampaio** [storybookjs/storybook#23633] [Help] [answered]
I was following the instructions in [here](https://storybook.js.org/addons/storybook-addon-playwright) but they were not working. After researching a lot for solutions to the problems I came across I found out the [addon repo](https://github.com/ccpu/storybook-addon-playwright) has been archived last month. I couldn't find any announcements about this, though. Hence my question, is the addon abandoned?

**Comment by shilman** [comment, marked as answer]
This addon isn't maintained by the storybook team, so you'd need to ask the author. If you want cross browser testing that's maintained, I'd recommend storybook's [test runner](https://storybook.js.org/docs/7.1/react/writing-tests/test-runner/) or [chromatic](https://www.chromatic.com/).

**Reply by bensampaio** [reply]
Ah thank you, it seems I was looking for the test runner!
